sdk/loadtestservice/azure-developer-loadtesting/azure/developer/loadtesting/aio/operations/_operations.py

```diff
@@ -1184,9 +1183,6 @@ def list_tests(
         :keyword last_modified_end_time: End DateTime(ISO 8601 literal format) of the last updated time
          range to filter tests. Default value is None.
         :paramtype last_modified_end_time: ~datetime.datetime
-        :keyword continuation_token_parameter: Continuation token to get the next page of response.
```

Comment:
is this change expected? if so it should be in the changelog too

Reply:
Yes this change is expected.